package.json

```
@@ -8,7 +8,7 @@
   "scripts": {
     "dev": "vite --host",
     "dev:watch": "cross-env WATCH=true vite --host",
-    "build": "tsc && cross-env NODE_ENV=production node --max_old_space_size=16384 ./node_modules/vite/bin/vite.js build --sourcemap true > build.txt",
+    "build": "tsc && cross-env NODE_ENV=development node --max_old_space_size=16384 ./node_modules/vite/bin/vite.js build --sourcemap true > build.txt",
```

Just a reminder to switch it back before merging the PR
```suggestion
    "build": "tsc && cross-env NODE_ENV=production node --max_old_space_size=16384 ./node_modules/vite/bin/vite.js build --sourcemap true > build.txt",
```